src/index.html

```diff
@@ -13,29 +13,24 @@
 </head>
 <body >
   <div class="wrapper">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <div class="container-fluid">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light opacity-50">
+        <div class="container-fluid mx-5">
           <div class='nav-logo me-3'></div>
```

Comment:
It's good to add a link for your logo, when clicked, to return users to the main page.